interfaces/acados_template/acados_template/generate_c_code_explicit_ode.py

```diff
@@ -0,0 +1,97 @@
+#   This file is part of acados.
```

Comment:
really cool to have this stuff also for python now!

Reply:
db345a5e56cffd71332e61a6def3266f779eac44 fixes this.